xblockgrademe/xblockgrademe.py

```diff
@@ -25,15 +25,29 @@ def workbench_scenarios():
             ('Xblock GradeMe',
              """<sequence_demo>
                     <xblockgrademe />
-                    <xblockgrademe name="My First XBlock" />
                 </sequence_demo>
              """),
         ]
 
-    name = String(
-        default='Xblock GradeMe',
+    display_name = String(
+        default="'Grade Me' Button",
         scope=Scope.settings,
-        help="This is the XBlock's name",
+        help="The name of the 'Grade Me' button",
+    )
+
+    description_text = String(
+        default=("<p>By clicking the button below, you assert that you have completed the course in its entirety. "
+                 "Your current score on the Progress page will be used to determine if you have earned a Statement of "
+                 "Accomplishment and for what level you qualify. Do not click the button below until you have "
+                 "finished the course and are satisfied with your score.</p>"),
+        scope=Scope.settings,
+        help="This is the description of what clicking the button means.",
+    )
+
+    button_text = String(
+        default='Yes, I have completed this course.',
+        scope=Scope.settings,
+        help="This is the text displayed on the button.",
```

Comment:
I'd prefer to wrap displayed/long strings in doubles. (in case we ever need to put apostrophes...)